From de77fa2d54cad7f1db21ae1ade470e4b40803102 Mon Sep 17 00:00:00 2001
From: Andreas Bombe <aeb@debian.org>
Date: Sat, 22 Nov 2014 15:12:12 +0100
Subject: [PATCH 1/2] Skip dh_auto_install, it installs binary into /usr/sbin

The standard target directory redirection attempted by dh_auto_install
is ignored by the upstream build framework and it installs efibootmgr
into /usr/sbin. Since the install step is not actually needed (the
files are installed directly from the build dir), simply omit
dh_auto_install instead of attempting to fix it.

Closes: 768889
---
 debian/rules | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/debian/rules b/debian/rules
index 2efe014..b587b86 100755
--- a/debian/rules
+++ b/debian/rules
@@ -8,6 +8,10 @@ export DH_OPTIONS=-v
 %:
 	dh $@
 
+# upstream build installs into /usr/sbin ignoring target directory;
+# since the install step is not actually needed just skip it
+override_dh_auto_install:
+
 override_dh_installman:
 	dh_installman
 
-- 
2.1.3

